src/include/miopen/hipoc_kernel.hpp

```diff
@@ -47,29 +48,20 @@ inline HipEventPtr make_hip_event()
 
 #if 1 // Keep around other storage techinques -- @pfultz2 27.03.2017
 
-#if 1 // Keep around other storage techinques -- @pfultz2 27.03.2017
```

Comment:
[R] I am not sure that we want to remove this. In my previous PR I decided to keep this as an illustration of the original idea.

Reply:
The original had a bug on line 54. The new code is the same as original except that line 54 is fixed. I don't think there's any benefit in keeping buggy code (rather may cause harm if someone assumes that it works).

Reply:
@amberhassaan I mean the following code:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/6d539ee81321121570606e4ef62e6d072775bbd8/src/include/miopen/hipoc_kernel.hpp#L65-L71
which was never used but explains the original idea.

Reply:
I don't see any value in keeping this code. It's just a simple pair class. We have always needed proper alignment for kernel arguments, so keeping around code that doesn't address alignment issues is just misguiding and confusing.

Reply:
Lets clean up the code, if we ever need it I am sure its not too difficult to write it again.

Reply:
@amberhassaan This comment is just a recommendation. You are free to either follow recommendations or deny them, no explanations needed.

> so keeping around code that doesn't address alignment issues is just misguiding and confusing.

Thanks for positive feedback.